packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -255,6 +257,7 @@ abstract class RenderSliverFixedExtentBoxAdaptor extends RenderSliverMultiBoxAda
     geometry = SliverGeometry(
       scrollExtent: estimatedMaxScrollOffset,
       paintExtent: paintExtent,
+      hitTestOrigin: hitTestOrigin,
```

Comment:
it is odd that we would need to shift the hitTestOrigin here, but the paint origin did not get shifted.